curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646b56e96902be0551c38b7a.md

```diff
@@ -0,0 +1,114 @@
+---
+id: 646b56e96902be0551c38b7a
+title: Step 20
+challengeType: 0
+dashedName: step-20
+---
+
+# --description--
+
+You will create an `img` HTML tag with the `src` attribute set to `an image link here`. Then, set the class name of the `img` to `select-icon`. This will be the dropdown icon.
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed